api-demos/dotnet/README.md

```diff
@@ -11,6 +11,7 @@
 1) Run `cd dotnet/YotiSignDemo`
 1) Rename `.env.example` to `.env`
 1) Add your Bearer Authentication Token to the `YOTI_AUTHENTICATION_TOKEN` variable in the `.env` file
-1) Customise the `options.json` file as you wish
-1) Run `dotnet run`
-1) visit <https://localhost:5001/sign>
+2) Set the API URL to match the production or demo environment: `/Controllers/SignController.cs L85` ;. Demo keys will not work in production and vice versa
```

Comment:
When contents are rendered in Markdown, you can put `1)` for every number, and then the markdown renderer will auto-increment them for you (you can see by visiting the [file on master](https://github.com/getyoti/yoti-sign-examples/blob/master/api-demos/dotnet/README.md)).

This means if you take one out, or add another one, usually you might forget to update the numbers, but using all `1)`s solves this problem :+1: